core/templates/dev/head/pages/exploration_editor/exploration_save_and_publish_buttons_directive.html

```diff
@@ -29,7 +29,7 @@
     </div>
   </li>
 
-  <li ng-if="isEditableOutsideTutorialMode()">
```

Comment:
This will show the "Publish" button to every guest/user. This is not expected.

Reply:
So, I’m confused as to how the help modal will flow. After the preview stage, should I direct the user (non-registered) to final message ? And the registered user to save button ?

Reply:
So, I’m confused as to how the help modal will flow. After the preview stage, should I direct the user (non-registered) to final message ? And the registered user to save button ?